atum/src/test/scala/za/co/absa/atum/AtumImplicitsSpec.scala

```diff
@@ -63,4 +63,71 @@ class AtumImplicitsSpec extends AnyFlatSpec with SparkTestBase with Matchers {
     spark.disableControlMeasuresTracking()
   }
 
+  "method getControlMeasure" should "return ControlMeasure object" in {
+    fs.delete(new Path(outputPath), false)
+
+    // Initializing library to hook up to Apache Spark
+    spark.enableControlMeasuresTracking(Some(inputPath), None)
+      .setControlMeasuresWorkflow("getControlMeasure")
+
+    import spark.implicits._
+    val df = spark.read.json(Seq(inputData).toDS)
+
+    // act
+    val controlMeasure = df.getControlMeasure
+
+    // assert
+    assert(controlMeasure == inputControlMeasure)
+
+    fs.delete(new Path(outputPath), false)
+    spark.disableControlMeasuresTracking()
+  }
+
+  "method getAllAdditionalInfo" should "return Map[String, String] with given info" in {
+    fs.delete(new Path(outputPath), false)
```

Comment:
Btw. since you are not writing to this file, maybe you do not need to attempt to delete it (all 3 accessor method tests, begin & end)

Reply:
Sure, I should have seen that.